docs/mesos/ManualInstallCalico.md

```diff
@@ -0,0 +1,65 @@
+# Manually Install the Calico Mesos Plugin
+This guide will walk you through manually downloading and installing the Calico-Mesos network plugin for your Mesos deployment. 
+
+The Calico-Mesos plugin must be installed with Calico's core services running on every Agent node in your cluster in order to take advantage of Calico networking. 
+
+*This guide assumes you already have [Mesos + Net-Modules](https://github.com/projectcalico/calico-docker/tree/master/docs/mesos#mesos--netmodules) installed on your Agent(s).* 
+
+> To obtain the Calico files, you will need `wget` installed. Please install the tool if you haven't already done so.
+
+## Download the Calico Mesos Plugin
+
+The Calico-Mesos plugin is available for download from the [calico-mesos repository releases](https://github.com/projectcalico/calico-mesos/releases). We recommend installing this binary in the `/calico` directory.
+
+    $ wget https://github.com/projectcalico/calico-mesos/releases/download/v0.1.1/calico_mesos
+    $ chmod +x calico_mesos
+    $ sudo mkdir /calico
+    $ sudo mv calico_mesos /calico/calico_mesos
+
+## Create the modules.json Configuration File
+To enable Calico networking in Mesos, you must create a `modules.json` file. When provided to the Mesos Agent process, this file will connect Mesos with the Net-Modules libraries as well as the Calico networking plugin, thus allowing Calico to receive Networking Events from Mesos.
+
+    $ cat > modules.json <<EOF
```

Comment:
I think its fine, IMO our guides are too nitpicky about sudo permission. The user will be able to figure out for themselves that the contents posted need to get to /calico/modules.json, by switching to sudo user or just editing it themselves